pkg/controller/utils/delayingreconciler.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"context"
+	log "github.com/sirupsen/logrus"
+	"regexp"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+const requeueDelay = time.Minute * 2
```

Comment:
I think we decided to make this the max backoff of the exponential rate limiter, which is [1000s](https://github.com/openshift/hive/blob/25f3291c14ccb736a6ccfe4ee1035359d7947565/pkg/controller/utils/utils.go#L147).